src/grid/PlanGridView.php

```diff
@@ -110,6 +110,16 @@ public function columns(): array
                     ],  
                 ]),
             ],
+            'parent_plan' => [
+                'format' => 'raw',
```

Comment:
When you set **format** to `raw` and do not encode output, XSS injection occurs.